RpcNep5Tracker/Nep5Transfer.cs

```diff
@@ -0,0 +1,55 @@
+using System.IO;
+using System.Numerics;
+using Neo.IO;
+using Neo.Ledger;
+using Neo.Network.P2P.Payloads;
+
+namespace Neo.Plugins
+{
+    public class Nep5Transfer : StateBase, ICloneable<Nep5Transfer>
+    {
+        public UInt160 UserScriptHash;
+        public uint BlockIndex;
+        public UInt256 TxHash;
+        public BigInteger Amount;
+
+        public override int Size => base.Size + 20 + Amount.ToByteArray().GetVarSize();
```

Comment:
`20 + 4 + 32+ Amount.ToByteArray().GetVarSize();` ?

Reply:
Your fix in #67 has been merged. Nice job.